examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>Can you use a try except clause when reading the data? There is an example at https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think just</p><pre class="ql-syntax" spellcheck="false">pd.read_csv(pm.get_data('../data/mastectomy.csv'))
</pre><p>should work fine</p>

Reply:
<p>Once this is merged, it could be changed to</p><p> </p><pre class="ql-syntax" spellcheck="false">pm.get_data('https://raw.githubusercontent.com/pymc-devs/pymc-examples/main/examples/data/mastectomy.csv')
</pre><p> </p>